Source/WebKit/UIProcess/WebProcessPool.h

```diff
@@ -41,6 +41,7 @@
 #include "WebsiteDataStore.h"
 #include <WebCore/CrossSiteNavigationDataTransfer.h>
 #include <WebCore/ProcessIdentifier.h>
+#include <WebCore/ProcessSwapDisposition.h>
```

Comment:
I think we should not have default values and we should forward declare ProcessSwapDisposition instead of including another header from a header and slowing down the build.  Same elsewhere.